Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,165 @@
+                moreEntries = false;
+            }
+
+            return moreEntries;
+        }
+
+
+        public static float CalculateBmi(float weightInKg, float height)
+        {
+            float bmi = weightInKg / (height * height);
+
+           
+            if((weightInKg <= 0) && (height <= 0))
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Weight cannot be equal or less than zero, but was {weightInKg}.");
+                Console.WriteLine($"Height cannot be less than zero, but was {height}.");
+                return -1;
+            }
+
+            if(weightInKg <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Weight cannot be equal or less than zero, but was {weightInKg}.");
+                return -1;
+            }
+
+            if(height <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Height cannot be equal or less than zero, but was {height}.");
+                return -1;
+            }
+
+            if(weightInKg >= height)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Weight cannot be more or equal to height.");
+                Console.WriteLine($" Height= {height}, Weight= {weightInKg}.");
+                return -1;
+            }
+
+            Console.WriteLine($"\nYour BMI based on your heigh of {height}m and your weight of {weightInKg}Kg " +
+                             $"is {bmi} kg/(m*m)");
+            return bmi;
+        }
+
+        //For Checks
+        public static int ConvertToInt(string message)
+        {
+            Console.WriteLine(message);
+
+            string toAnInt = Console.ReadLine();
+            
+
+            if (string.IsNullOrEmpty(toAnInt))
+            {
+                return 0;
+            }
+           
+
+            bool isInt = int.TryParse(toAnInt, out int age);
+
+            if(!isInt)
+            {
+                Console.Write($"\"{toAnInt}\" is not a valid number.");
+                return -1 ;
+            }
+
+            return age;
+        }
+
+
+        public static string ConvertToString(string message)
+        {
+            Console.WriteLine(message);
+
+            string readInString = Console.ReadLine();
+
+            
+           if(string.IsNullOrEmpty(readInString))
+            {
+                Console.Write($"Name cannot be empty.");
+                return "-";
+            }
+            
+            return readInString;
+        }
+
+
+        public static float ConvertToFloat(string message)
+        {
+            Console.WriteLine(message);
+
+            string toAFloat = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(toAFloat))
+            {
+               // Console.WriteLine("You did not enter anything.");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4871920)